packages/react/src/Textbox/Textbox.tsx

```diff
@@ -0,0 +1,17 @@
+import { Textbox as CommunityTextBox } from '@utrecht/component-library-react'
+import { ForwardedRef, forwardRef, InputHTMLAttributes } from 'react'
+
+export type TextboxTypes = 'number' | 'text'
```

Comment:
Reason not to add them, is because some of them also have visual differences. Like password, search. Also no designs and other specs. I guess this should go back for refinements

Reply:
We then can decide for naming as well